src/plugins/recordTypes/dutyofcare/fields.js

```diff
@@ -0,0 +1,337 @@
+import { defineMessages } from 'react-intl';
+
+export default (configContext) => {
+  const {
+    AutocompleteInput,
+    CompoundInput,
+    DateInput,
+    IDGeneratorInput,
+    TextInput,
+    TermPickerInput,
+  } = configContext.inputComponents;
+
+  const {
+    configKey: config,
+  } = configContext.configHelpers;
+
+  const {
+    DATA_TYPE_DATE,
+  } = configContext.dataTypes;
+
+  const {
+    extensions,
+  } = configContext.config;
+
+  const {
+    validateNotInUse,
+  } = configContext.validationHelpers;
+
+  return {
+    document: {
+      [config]: {
+        view: {
+          type: CompoundInput,
+          props: {
+            defaultChildSubpath: 'ns2:dutyofcares_common',
+          },
+        },
+      },
+      ...extensions.core.fields,
+      'ns2:dutyofcares_common': {
+        [config]: {
+          service: {
+            ns: 'http://collectionspace.org/services/dutyofcare',
+          },
+        },
+        dutyOfCareNumber: {
+          [config]: {
+            cloneable: false,
+            messages: defineMessages({
+              inUse: {
+                id: 'field.dutyofcares_common.dutyOfCareNumber.inUse',
+                defaultMessage: 'The identification number {value} is in use by another record.',
+              },
+              name: {
+                id: 'field.dutyofcares_common.dutyOfCareNumber.name',
+                defaultMessage: 'Duty of care id',
```

Comment:
@ray-lee I updated the casing for this but just unresolved this - we don't really use ID that much in the forms and normally use `Identification/Reference/etc number` instead. I know for some of the others I had used that instead of `ID`, and I'm thinking of updating this as well to be `Duty of care number` to align better with other records.